include/map_updater.h

```diff
@@ -10,6 +10,6 @@ class MapUpdater
 public:
   static void MoveLinesDown(int ticks, char* game_map, char* new_line);
 };
-} // namespace roadrun
+}  // namespace roadrun
 
 #endif
```

Comment:
maybe add a newline at the bottom of this file

Reply:
done. Changes like this could probably be easily suggested through [the builtin suggestion tool](https://haacked.com/archive/2019/06/03/suggested-changes/).

More info [here](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request) and [here](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request)